### ISSUE TITLE
gitk GUI doens't show diff view at bottom.

### ISSUE THREAD

**Issue by indigo-sam** [git-for-windows/git#1558]
Git: 2.16.2
Windows 10 64x
I don't see the highlighted DIFF view anymore in the UI opened from gitk command. Anyone know if this is  a bug or an option that I can enable somewhere?

![image](https://user-images.githubusercontent.com/20443379/37419429-b25fbfce-2771-11e8-8ba7-30f52854de0d.png)

**Comment by dscho** [comment]
Is this a public repository?

(BTW please do not say "Git GUI" when you mean "gitk"; they are different commands with very different UIs).

**Comment by dscho** [comment]
Oh, and: do you see the diff when you scroll down? (There is a scroll bar between the left and the right pane, and it seems to control the left pane.)

**Comment by indigo-sam** [comment]
I can scroll up(blank space), but can't scroll down since. And there isn't a scroll bar between left and right pane either.

**Comment by indigo-sam** [comment]
This is what I am seeing. 
![image](https://user-images.githubusercontent.com/20443379/37431347-37929912-2792-11e8-9fdf-7a6dc5f8f32a.png)

**Comment by dscho** [comment]
@indigo-sam you have me completely confused. Is the screenshot you were showing in the bug report *not* what you are seeing? I see a very clear scroll bar there, in the part you circled. Your most recent screenshot does not show that part. So is the latter what you are seeing? If so, this is a "funny" behavior of `gitk`: if you hover near the bottom, you should see the cursor change to a vertical double-arrow, and be able to "unhide" the bottom half that obviously was moved out of view in favor of more screen estate for the upper half.

**Comment by indigo-sam** [comment]
Hi dscho, as I said in the description, the highlighted section(everything inside blue line) from image doesn’t not show. So there isn’t a vertical scroll bar. The latter image is what I am seeing. I have been trying to hover to the bottom many times, the double-arrow never show up. Thus I created this question/bug.  I have been using gitk very day for many years. Is there a way to reset the layout ? I tried to reinstall it but problem is still there.  To answer your first reply, this is a private repo.

**Comment by dscho** [comment]
@indigo-sam no, nothing of the likes changes in `gitk`. Can you resize the window vertically? Maybe that made the lower panel invisible?

**Comment by indigo-sam** [comment]
Hi dscho, as I said in the description, the highlighted section(everything inside blue line) from image doesn’t not show. So there isn’t a vertical scroll bar. The latter image is what I am seeing. I have been trying to hover to the bottom many times, the double-arrow never show up. Thus I created this question/bug.  I have been using gitk very day for many years. Is there a way to reset the layout ? I tried to reinstall it but problem is still there.  To answer your first reply, this is a private repo.
